pkgs/development/libraries/SDL2/default.nix

```diff
@@ -39,6 +40,7 @@ stdenv.mkDerivation rec {
 
   # Since `libpulse*.la' contain `-lgdbm', PulseAudio must be propagated.
   propagatedBuildInputs = lib.optionals x11Support [ libICE libXi libXScrnSaver libXcursor libXinerama libXext libXrandr libXxf86vm ] ++
+    lib.optional waylandSupport [ wayland wayland-protocols libxkbcommon.dev ] ++
```

Comment:
This line adds the single value `[ wayland wayland-protocols libxkbcommon.dev ]` to `propagatedBuildInputs`, not those three. It should use `lib.optionals`, like:

````
lib.optionals waylandSupport [ wayland wayland-protocols libxkbcommon ]
````

Reply:
@dezgeg Indeed, thank you!